pkg/products/rhssocommon/reconciler.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	monitoringv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
 	grafanav1alpha1 "github.com/integr8ly/grafana-operator/v3/pkg/apis/integreatly/v1alpha1"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Comment:
Looks good ! :+1: Though I think it would be good to use `v1` of `apiextensions` to align with this PR https://github.com/integr8ly/integreatly-operator/pull/1685 :thinking:

Reply:
I've tried `v1` and `apiextensions` and none of them was recognized by the live cluster. It seemed odd once I found this line in the CRD `apiVersion: apiextensions.k8s.io/v1`  
The error I"m getting while trying with anything but v1beta1: `no kind is registered for the type v1.CustomResourceDefinition in scheme \"pkg/runtime/scheme.go:101\"` :thinking:

Reply:
Sounds like maybe https://github.com/integr8ly/integreatly-operator/blob/master/apis/v1alpha1/addtoscheme_integreatly_v1alpha1.go#L57-L92 needs to be added to / updated to work with `v1` :thinking:

Reply:
isn't it already? https://github.com/integr8ly/integreatly-operator/blob/master/apis/v1alpha1/addtoscheme_integreatly_v1alpha1.go#L92 :thinking:

Reply:
`v1beta1` is but `v1` is not from what I can see, which might be why it didn't work when changed to `v1` :thinking: 
(side note: If this does work using `v1`, would be good to use `apiextensionv1` as the import name to follow #1685)

Reply:
sorry for being so silly. Updated and it was tested on a cluster. Also updated unit tests to work with v1